scripts/rmc_flags.py

```diff
@@ -139,6 +139,16 @@ def add_check_flags(make_group, add_flag, config):
     add_flag(group, "--unwinding-checks", default=True, action=BooleanOptionalAction,
              help="Turn on default unwinding checks")
 
+# Add flags for common CBMC flags
+def add_common_flags(make_group, add_flag, config):
+    group = make_group("Common flags", "Common CBMC flags handled by RMC.")
+    add_flag(group, "--object-bits", type=str,
+             help="Specify the number of bits used for representing object IDs in CBMC")
+    add_flag(group, "--unwind", type=str,
+             help="Specify the value used for loop unwinding in CBMC")
+    add_flag(group, "--auto-unwind", default=False, action=BooleanOptionalAction,
```

Comment:
Isn't this technically `default=True` since this is the current default behavior?

Reply:
This depends very much on the `--unwind 0` by default discussion.

What `--auto-unwind` does is prevent `--unwind` flags from being used anywhere by raising an exception. We could change it so it removes `--unwind` flags instead and enable it by default. But note that if we set this to true as is, users would need to disable this (by passing `--no-auto-unwind`) if they pass the unwinding flag anywhere.

Reply:
Makes sense.